applications/registration-api/src/test/java/no/dcat/controller/ApiRegistrationControllerTest.java

```diff
@@ -0,0 +1,99 @@
+package no.dcat.controller;
```

Comment:
Bare enhetstest de endepunkter som inneholder noe logikk.
De glue-endepunkter skulle bare ha IT.

Så hvis du klarer å flytter funksionalite ut av endepunkt-funksjoner, son at gjenstår bare glue, da trenger du ikke ehnetstest controller!